modules/wpml/wpml-config.php

```diff
@@ -138,6 +142,7 @@ public function init() {
 		add_filter( 'pll_term_meta_encodings', array( $this, 'add_term_meta_encodings' ), 20 );
 		add_filter( 'pll_blocks_xpath_rules', array( $this, 'translate_blocks' ) );
 		add_filter( 'pll_blocks_rules_for_attributes', array( $this, 'translate_blocks_attributes' ) );
+		add_filter( 'pll_block_attribute_encodings', array( $this, 'decode_blocks_attributes' ) );
```

Comment:
We are not consistent with all filters. Some have the priority 10, others 20. It would make sense to always use 20 (the idea is that a custom wpml-config can override a filter at the default priority). What do you think?

Reply:
Without thinking specifically to hook priority, that's a question I asked myself at some point indeed: should the file overwrite a filter hook, or the opposite? Now I have the answer.